subprojects/packagefiles/asio/meson.build

```diff
@@ -19,3 +19,5 @@ asio_dep = declare_dependency(
     include_directories : include_directories('include'),
     dependencies: asio_deps,
 )
+
+install_headers('include/asio.hpp', 'include/asio/*')
```

Comment:
```meson
install_headers('include/asio.hpp')
install_subdir('include/asio', install_dir: get_option('includedir'))
```

This is because install_headers() takes files, not directories. :(